src/pages/ListScholarships.jsx

```diff
@@ -96,13 +97,27 @@ function ListScholarships() {
     } else pruneQueryParam(qParams.GRADES);
   }
 
+  /**
+   * prunes inalid major values not present in the MAJORS set
```

Comment:
No need to prune as we allow freeform major values.

Reply:
got it.
just pruned if `majors` is not an array/empty array